docs/installation/index.md

```
@@ -165,7 +165,54 @@ pip install deffcode-0.2.0-py3-none-any.whl
+
+??? info "Don't have Poetry installed?"
+
+    Follow the [official Poetry installation guide](https://python-poetry.org/docs/#installation) before proceeding. You can verify your install with:
+
+    ```sh
+    poetry --version
+    ```
+
+**Add DeFFcode to an existing Poetry project:**
+
+```sh
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)